src/cmd/linuxkit/pkglib/pkglib.go

```diff
@@ -169,6 +178,36 @@ func NewFromCLI(fs *flag.FlagSet, args ...string) (Pkg, error) {
 		}
 	})
 
+	var srcHashes string
+	sources := []pkgSource{pkgSource{src: pkgPath, dst: "/"}}
+
+	for _, source := range pi.ExtraSources {
+		tmp := strings.Split(source, ":")
+		if len(tmp) != 2 {
+			return Pkg{}, fmt.Errorf("Bad source format in %s", source)
+		}
+
+		srcPath := tmp[0]
```

Comment:
An absolute `srcPath` is never cleaned AFAICT (a relative one gets it implicitly in `filepath.Join`).